advisories/hackage/biscuit-haskell/HSEC-0000-0000.md

```diff
@@ -0,0 +1,45 @@
+```toml
+[advisory]
+# Identifier for the advisory (mandatory). Will be assigned a "HSEC-YYYY-NNNN"
+# identifier e.g. HSEC-2022-0001. Please use "HSEC-0000-0000" in PRs.
+id = "HSEC-0000-0000"
+
+# Publication date of the advisory as an RFC 3339 date.
+# DO NOT INCLUDE THIS in files committed to Git.
+# It will be derived from the Git commit history.
+# date = 2024-07-31
```

Comment:
Please drop these comment lines (and other comments explaining the format)

Reply:
done